### ISSUE TITLE
Missing ClusterRole.AggregationRule

### ISSUE THREAD

**Issue by curtbushko** [sl1pm4t/k2tf#81]
k2tf --version                                                                                                          
k2tf version: 0.5.0

When running k2tf against some yaml I am getting the warning of:

```
4:32PM Warn | excluding attribute [kubernetes_cluster_role.aggregation_rule] not found in Terraform schema  field=ClusterRole.AggregationRule name=source_observer type=kubernetes_cluster_role
```

Is it possible to get a schema update for this extremely valuable tool?

**Comment by sl1pm4t** [comment]
Hi @curtbushko - yes a schema update is long over due. I'll try work on this soon.

**Comment by sl1pm4t** [comment]
Fixed by #85 and #86

**Comment by curtbushko** [comment]
@sl1pm4t Thanks so much for fixing this! It is very much appreciated!
